hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/AbstractDelegationTokenSecretManager.java

```diff
@@ -766,6 +774,10 @@ private void removeExpiredToken() throws IOException {
     logExpireTokens(expiredTokens);
   }
 
+  protected Map<TokenIdent, DelegationTokenInformation> getTokensForCleanup() {
```

Comment:
Maybe rename to `getCandidateTokensForCleanup()` so it is clear there are other checks that will happen.

Reply:
Makes sense, I've renamed it